packages/fastauth-metamask/src/lib/lib.tsx

```diff
@@ -0,0 +1,399 @@
+import React from 'react'
```

Comment:
Is this file needed, how do we make use of this?

There are some linting issues with the JSX, try running:
 `yarn nx run fastauth-metamask:lint`